.semaphore/semaphore.yml

```diff
@@ -152,8 +152,10 @@ blocks:
       matrix:
       - env_var: ARCH
         values: [ "arm64", "ppc64le" ]
+      - env_var: RELEASE_LDFLAGS
+        values: [ "-w -s" ]
       commands:
-      - ../.semaphore/run-and-monitor image-$ARCH.log make image ARCH=$ARCH
+      - ../.semaphore/run-and-monitor image-$ARCH.log make image ARCH=$ARCH RELEASE_LDFLAGS=$RELEASE_LDFLAGS
```

Comment:
I still think we shouldn't be touching semaphore.yaml at all - we want the dev builds (the ones produced by this file) to include symbols.